src/apps/activities/domain/response_values.py

```diff
@@ -329,6 +337,34 @@ class AudioPlayerValues(PublicModel):
     file: str | None = Field(default=None)
 
 
+class _PhrasalTemplateTextField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.SENTENCE
+    text: str
+
+
+class _PhrasalTemplateItemResponseField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.ITEM_RESPONSE
+    id: str | None = None
+    display_mode: str
```

Comment:
Should we predefine values for this? Checking with product/design to see what they may be.

Reply:
Maybe `display_mode` should be optional since some of these item responses don't support a display mode ([item response type 1](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=293-193762&t=WX7ykB8LkJJMeeDP-4)).

@mbanting I agree about the predefined values, and I think we should also add an optional `display_mode_order` field as well. Here are my suggestions based on what's in [figma](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=355-54173&t=WX7ykB8LkJJMeeDP-4):

```python
class DisplayMode(str, Enum):
    BULLET_LIST = "bullet_list"
    SENTENCE = "sentence"
    RESPONSE_FROM_SLIDER = "response_from_slider"
    
class DisplayModeOrder(str, Enum):
    OPTION_TEXT_ROW_TEXT = "option_text-row_text"
    ROW_TEXT_OPTION_TEXT = "row_text-option_text"

```

Let me know if this makes sense

Reply:
added